data/creaturescripts/scripts/playerdeath.lua

```diff
@@ -1,8 +1,9 @@
 local deathListEnabled = true
 local maxDeathRecords = 5
 
-function onDeath(player, corpse, killer, mostDamageKiller, unjustified, mostDamageUnjustified)
```

Comment:
if its registered only for player then this parameter will be always player, so why changing it?